tools/bowtie2.wdl

```diff
@@ -0,0 +1,447 @@
+            ~{if defined(read_group.DT) then "--rg DT:~{read_group.DT}" else ""} \
+            ~{if defined(read_group.FO) then "--rg FO:~{read_group.FO}" else ""} \
+            ~{if defined(read_group.KS) then "--rg KS:~{read_group.KS}" else ""} \
+            ~{if defined(read_group.LB) then "--rg LB:~{read_group.LB}" else ""} \
+            ~{if defined(read_group.PG) then "--rg PG:~{read_group.PG}" else ""} \
+            ~{if defined(read_group.PI) then "--rg PI:~{read_group.PI}" else ""} \
+            ~{if defined(read_group.PL) then "--rg PL:~{read_group.PL}" else ""} \
+            ~{if defined(read_group.PM) then "--rg PM:~{read_group.PM}" else ""} \
+            ~{if defined(read_group.PU) then "--rg PU:~{read_group.PU}" else ""} \
+            ~{if defined(read_group.SM) then "--rg SM:~{read_group.SM}" else ""} \
+            ~{if defined(addl_rg_text) then "--rg ~{addl_rg_text}" else ""} \
+            ~{if omit_sec_seq then "--omit-sec-seq" else ""} \
+            ~{if sam_no_qname_trunc then "--sam-no-qname-trunc" else ""} \
+            ~{if xeq then "--xeq" else ""} \
+            ~{if soft_clipped_unmapped_tlen then "--soft-clipped-unmapped-tlen" else ""} \
+            ~{if sam_append_comment then "--sam-append-comment" else ""} \
+            ~{if defined(sam_opt_config) then "--sam-opt ~{sam_opt_config}" else ""} \
+            -p ~{ncpu} \
+            ~{if reorder then "--reorder" else ""} \
+            ~{if qc_filter then "--qc-filter" else ""} \
+            --seed ~{seed} \
+            --time \
+            ~{if non_deterministic then "--non-deterministic" else ""} \
+            ~{"--score-min ~{score_min.function_type},~{score_min.constant},~{score_min.coefficient}"} \
+            ~{"-i ~{interval_seed_substrings.function_type},~{interval_seed_substrings.constant},~{interval_seed_substrings.coefficient}"} \
+            ~{"--n-ceil ~{n_ceil.function_type},~{n_ceil.constant},~{n_ceil.coefficient}"} \
+            -D ~{max_failed_extends} \
+            -R ~{repetitive_seeds} \
+            -x bowtie_db/"$PREFIX" \
+            ~{(
+                if defined(read_two_fastq_gz)
+                then "-1 ~{read_one_fastq_gz}"
+                else "-U ~{read_one_fastq_gz}"
+            )} \
+            ~{if defined(read_two_fastq_gz) then "-2 ~{read_two_fastq_gz}" else ""} \
+            | samtools view -bS - > ~{prefix}.bam    
+    >>>
+
+    output {
+        File aligned_bam = prefix + ".bam"
+        File? unpaired_unaligned = prefix + ".unpaired_unaligned.fastq.gz"
+        File? unpaired_aligned = prefix + ".unpaired_aligned.fastq.gz"
+        File? paired_discordant_read_one = prefix + ".paired_discordant.fastq.1.gz"
+        File? paired_discordant_read_two = prefix + ".paired_discordant.fastq.2.gz"
+        File? paired_concordant_read_one = prefix + ".paired_concordant.fastq.1.gz"
+        File? paired_concordant_read_two = prefix + ".paired_concordant.fastq.2.gz"
```

Comment:
Looks like this naming convention is coming from `bowtie`? Will it be properly handled by that regex we use for getting prefixes out of FASTQ names?

Reply:
It is from bowtie2. I doubt our regex will handle these (but I didn't look). That said, these reads shouldn't be passed into anything downstream, at least in our code.